src/views/Benchmark/index.jsx

```diff
@@ -1,22 +1,21 @@
 import PropTypes from 'prop-types';
 import { Component } from 'react';
-import { withStyles } from '@material-ui/core/styles';
 import MetricsGraphics from 'react-metrics-graphics';
-
 import { curveLinear } from 'd3';
 import { subbenchmarksData } from '@mozilla-frontend-infra/perf-goggles';
+import { withRouter } from 'react-router-dom';
 import Header from '../../components/Header';
 import CONFIG from '../../config';
 import prepareData from '../../utils/prepareData';
 import './benchmark.css';
 
-const styles = () => ({
-  root: {},
-});
-
 class Benchmark extends Component {
   static propTypes = {
-    classes: PropTypes.shape().isRequired,
+    benchmark: PropTypes.string.isRequired,
+    history: PropTypes.shape({
```

Comment:
I typically don't declare prop types for props passed by HOCs I don't own, since they will be guaranteed to be passed by the HOC. Feel free to remove this prop type.